test/e2e/apps/statefulset.go

```diff
@@ -843,6 +847,75 @@ var _ = SIGDescribe("StatefulSet", func() {
 				return nil
 			}, framework.StatefulPodTimeout, 2*time.Second).Should(BeNil())
 		})
+
+		It("should have a working scale subresource", func() {
```

Comment:
@liggitt Thank you very much for your detailed explanation! i will dive deep into each of the components carefully, and get back to you tomorrow. Thanks!